misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/distribution/DistributionStepDropToScreen.jsx

```diff
@@ -68,15 +99,17 @@ const mapStateToProps = (state, ownProps) => {
   const lineProps = activity != null ? activity.dataStored.lines[lineId] : null;
   const stepProps = lineProps != null && lineProps.steps ? lineProps.steps[stepId] : {};
 
-  console.log('stepProps', stepProps);
-
   return {
     wfProcessId,
     activityId,
     lineId,
     stepId,
-    locatorBarcode: stepProps.dropToLocator.barcode,
+    caption: stepProps.dropToLocator.caption,
```

Comment:
would extract those and add safety checks. What would happen if dropLocator is undefined ?

Reply:
IMHO that's ok. `dropLocator` shall always be provided.